cli.js

```diff
@@ -30,7 +32,13 @@ if (!username) {
 
 	const createRow = (prefix, key) => {
 		if (user[key]) {
-			rows.push(`${prefix}: ${user[key]}`);
+			if (key === 'github') {
+				rows.push(`${chalk.green(prefix)}: ${chalk.bold("@"+user[key])} ${chalk.gray(terminalLink(`(Click Here)`, `https://github.com/${user[key]}`))}`);
```

Comment:
The actual username should be the link. Not a "Click here" text.

Reply:
I modify as soon as possible